React/Modules/RCTStatusBarManager.m

```diff
@@ -49,17 +49,15 @@ static BOOL RCTViewControllerBasedStatusBarAppearance()
 
 - (NSArray<NSString *> *)supportedEvents
 {
-  return @[@"statusBarFrameDidChange",
-           @"statusBarFrameWillChange"];
+  return @[@"statusBarFrameDidChange"];
 }
 
 #if !TARGET_OS_TV
 
 - (void)startObserving
 {
   NSNotificationCenter *nc = [NSNotificationCenter defaultCenter];
-  [nc addObserver:self selector:@selector(applicationDidChangeStatusBarFrame:) name:UIApplicationDidChangeStatusBarFrameNotification object:nil];
-  [nc addObserver:self selector:@selector(applicationWillChangeStatusBarFrame:) name:UIApplicationWillChangeStatusBarFrameNotification object:nil];
+  [nc addObserver:self selector:@selector(applicationDidChangeStatusBarFrame) name:UIApplicationDidChangeStatusBarFrameNotification object:nil];
```

Comment:
Why did we remove `willChange` event?

Reply:
We can't subtract the y offset of the root view in the willChange event so we can't calculate the height of the portion of the status bar people actually care about from within that event.